gradle/mockito-core/inline-mock.gradle

```diff
@@ -0,0 +1,13 @@
+task copyMockMethodDispatcher(type: Copy) {
+    from "${buildDir}/classes/main/org/mockito/internal/creation/bytebuddy"
+    into "${buildDir}/classes/main/org/mockito/internal/creation/bytebuddy"
+    include "MockMethodDispatcher.class"
+    rename { String fileName ->
+        fileName.replace('.class', '.raw')
+    }
+}
+classes.dependsOn(copyMockMethodDispatcher)
```

Comment:
Yes, for the time until packaging the jar. This is required to make IDEs work with Mockito which do otherwise claim that there is a class missing (experienced with IntellIJ). It does not hurt us as we know that our unit tests are run by a parent-first class loader where class loading works as expected. This is only an issue when dealing with parent-first loaders where the `.class` file must not exist.